NOTICES

```diff
@@ -0,0 +1,2022 @@
+      agreed to in writing, Licensor provides the Work (and each
+      Contributor provides its Contributions) on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+      implied, including, without limitation, any warranties or conditions
+      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+      PARTICULAR PURPOSE. You are solely responsible for determining the
+      appropriateness of using or redistributing the Work and assume any
+      risks associated with Your exercise of permissions under this License.
+
+   8. Limitation of Liability. In no event and under no legal theory,
+      whether in tort (including negligence), contract, or otherwise,
+      unless required by applicable law (such as deliberate and grossly
+      negligent acts) or agreed to in writing, shall any Contributor be
+      liable to You for damages, including any direct, indirect, special,
+      incidental, or consequential damages of any character arising as a
+      result of this License or out of the use or inability to use the
+      Work (including but not limited to damages for loss of goodwill,
+      work stoppage, computer failure or malfunction, or any and all
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+
+=> Flake8
+Copyright (C) 2011-2013 Tarek Ziade <tarek@ziade.org>
+Copyright (C) 2012-2016 Ian Cordasco <graffatcolmingov@gmail.com>
+=> Hyperium hyper for Rust
+Copyright (c) 2014 Sean McArthur
+=> JSON-C
+Copyright (c) 2009-2012 Eric Haszlakiewicz
+Copyright (c) 2004, 2005 Metaparadigm Pte Ltd
+=> Joe Hildebrand node-cbor
+Copyright (c) 2014 Joe Hildebrand
+=> Node.js
+Copyright Node.js contributors. All rights reserved
+Copyright Joyent, Inc. and other Node contributors. All rights reserved.
+=> PyYAML - a YAML parser and emitter for Python
+Copyright (c) 2006 Kirill Simonov
+=> aiodns
+Copyright (C) 2014 by Saúl Ibarra Corretgé
+=> cffi
+Copyright (C) 2005-2007, James Bielman  <jamesjb@jamesjb.com
+=> cppzmq
```

Comment:
This one seems missing the who: https://github.com/zeromq/cppzmq/blob/master/LICENSE.

Reply:
Although it is an MIT license, the author has not chosen to include a copyright notice in the license file, and so I have not include a copyright notice in this notices file.  I  am trying to make the artifacts in this file match the project licence artifacts as closely as possible while minimizing duplication.